src/Infrastructure/Persistence/Doctrine/Migrations/Version20230315101340.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230315101340 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE measure (uuid UUID NOT NULL, location_uuid UUID NOT NULL, type VARCHAR(255) NOT NULL, PRIMARY KEY(uuid))');
```

Comment:
cf autre commentaire

```suggestion
        $this->addSql('CREATE TABLE measure (uuid UUID NOT NULL, location_uuid UUID NOT NULL, type VARCHAR(16) NOT NULL, PRIMARY KEY(uuid))');
```